source/administration/changelog.md

```diff
@@ -4,6 +4,153 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
+ - Due to fixing performance issues related to emoji reactions, the performance of the upgrade has been affected in that the schema upgrade now takes more time in environments with lots of reactions in their database. These environments are recommended to perform the schema migration during low usage times and potentially in advance of the upgrade. Since this migration happens before the Mattermost Server is fully launched, non-High Availability installs will be unreachable during this time.
+ - On mobile apps, users will not be able to see LDAP group mentions (E20 feature) in the autocomplete dropdown. Users will still receive notifications if they are part of an LDAP group. However, the group mention keyword will not be highlighted.
+ 
+**IMPORTANT:** If you upgrade from a release earlier than 5.23, please read the other [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html).
+
+### Highlights
+
+#### Notify AD/LDAP Groups with a single @mention (Beta) (E20) 
+ - Ability to enable mentions for LDAP-synced groups so users can notify the entire group at the same time.
+
+#### Manage users from the System Console (E20)
+ - Ability to view and manage members via each team or channel configuration page.
+
+#### Sync profile images from AD/LDAP (E10) 
+ - Ability to ensure compliance with corporate policies by automatically syncing profile images from AD/LDAP.
+
+#### Automatically extending user sessions
+ - Ability to enable a feature that automatically extends session lengths when users are active on Mattermost apps.
+
+#### Access CLI remotely
+ - Ability to manage Mattermost without having direct access to the server with a new Local Mode for mmctl.
+
+#### Improved search experience
+ - Ability to use the mouse or keyboard to select search filters instead of typing them manually.
+ 
+#### Slash command autocomplete framework (Beta)
+ - Ability to make slash commands easier to use and increase discoverability with a new slash command autocomplete framework for plugins.
+
+#### Full-text search and indexing (Experimental)
+ - Ability to use Bleve to execute search functionality instead of the database.
+
+### Improvements
+
+#### Enterprise Edition (EE)
+ - All Enterprise Edition features are now disabled after the grace period is over instead of only after a server restart. Also, the grace period was changed from 15 to 10 days. Please see https://mattermost.com/pricing/#faq for more details.
```

Comment:
```suggestion
 - Grace period after Enterprise Edition subscription expires was reduced from 15 days to 10 days. Moreover, Enterprise features are now disabled immediately after the grace period is over, instead of only after a server restart. Please see https://mattermost.com/pricing/#faq for more details.
```